gtsam/sfm/ShonanAveraging.h

```diff
@@ -53,6 +53,8 @@ struct GTSAM_EXPORT ShonanAveragingParameters {
   double alpha;                 // weight of anchor-based prior (default 0)
   double beta;                  // weight of Karcher-based prior (default 1)
   double gamma;                 // weight of gauge-fixing factors (default 0)
+  bool useHuber;  // if enabled, the Huber loss is used in the optimization
```

Comment:
Spacing for comments is inconsistent. (Format changed part only)

Reply:
This is the result of the formatter.